src/Sulu/Component/Webspace/Analyzer/Attributes/PortalInformationRequestProcessor.php

```diff
@@ -114,6 +114,10 @@ private function getResourceLocatorFromRequest(PortalInformation $portalInformat
             strlen($portalInformation->getUrl())
         );
 
+        if ('php' === $formatResult) {
```

Comment:
Shouldn't we fallback to `html` in general? We have exactly the same problem when people are typing something like `/index.asdf`.

Reply:
@danrot 
@wachterjohannes or @chirimoya said there where usecases where the ending was something like `.newsletter` so this would make it a little difficult without a bc break doing that.

Reply:
@danrot maybe instead of overwriting the request format. we should always render the when symfony exception controller return html: https://github.com/sulu/sulu/blob/1fee9e07b9c16e808c87974ef6a92c1da6dd4ba8/src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php#L92?

Reply:
@danrot maybe we should just fix something in the exception controller instead of overwritting the request format behaviour? /cc @sulu/core-team

Reply:
@alexander-schranz Your latest suggestion what we have just discussed with @chirimoya :-) I think we should do it this way. This keeps the old behaviour and fixes the messed up error message for unknown formats.